LocalSettings.php

```diff
@@ -381,6 +381,28 @@
 	'wgCheckUserEnableSpecialInvestigate' => [
 		'default' => true,
 	],
+	'wgCheckUserLogLogins' => [
+		'default' => true,
+	],
+	'wgCheckUserCAtoollink' => [
```

Comment:
For all three of these options, the wiki should be metawiki, and the group should be steward. The wiki value is used only to generate a link to the special page on that wiki, and all our global (b)locks are done through metawiki. For the group, it tries to check the global group, as long as CentralAuth is installed.

Reply:
Thanks! Done.